From de88443ebcc892c858f7aee32357183f1cf68f09 Mon Sep 17 00:00:00 2001
From: Andre Przywara <andre.przywara@arm.com>
Date: Thu, 3 Oct 2024 00:30:34 +0100
Subject: [PATCH 01/29] dt-bindings: irq: sun7i-nmi: document the Allwinner
 A523 NMI controller

The Allwinner A523 SoC contains an NMI controller compatible to one used
in the recent Allwinner SoCs.

Add the A523 specific name to the list of allowed compatible strings.

Signed-off-by: Andre Przywara <andre.przywara@arm.com>
---
 .../interrupt-controller/allwinner,sun7i-a20-sc-nmi.yaml         | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/interrupt-controller/allwinner,sun7i-a20-sc-nmi.yaml b/Documentation/devicetree/bindings/interrupt-controller/allwinner,sun7i-a20-sc-nmi.yaml
index f49b43f45f3..d4a86694a03 100644
--- a/Documentation/devicetree/bindings/interrupt-controller/allwinner,sun7i-a20-sc-nmi.yaml
+++ b/Documentation/devicetree/bindings/interrupt-controller/allwinner,sun7i-a20-sc-nmi.yaml
@@ -31,6 +31,7 @@ properties:
               - allwinner,sun8i-v3s-nmi
               - allwinner,sun50i-a100-nmi
               - allwinner,sun50i-h616-nmi
+              - allwinner,sun55i-a523-nmi
           - const: allwinner,sun9i-a80-nmi
 
   reg:
-- 
2.46.0

